the_encryptor_v2.py

```diff
@@ -22,20 +22,13 @@ def goodbye():
 		bye += "."
 
 key = ""
-skip_files = ["the_encryptor_v2.py", ".keyfile"]
+skip_files = ["the_encryptor_v2.py", ".keyfile", ".gitignore"]
```

Comment:
we could add readme here, or better yet we could have it look in a special folder and/or have the user select a folder within the current directory... so many options